### ISSUE TITLE
No notifications on android

### ISSUE THREAD

**Issue by mrnswetland** [balzack/databag#134]
I am running the latest installation in docker and don't get notifications on my android device. My partner has an android as well and doesn't receive notifications either. The setting it turned on both in the admin settings and the account settings.

**Comment by mrnswetland** [comment]
The android app we are using is the one from the Google play store

**Comment by balzack** [comment]
Hello, sorry for the issues you are seeing. Could you tell me which android phones you and your partner are using? Thank you.

**Comment by mrnswetland** [comment]
Samsung Galaxy S24, Nuu A23, and Samsung A32

**Comment by Stellarus37** [comment]
I have the same problem, i'm using a Xiaomi MI10T with lineageOS 21, and same with a Samsung. But I'm using Cloudflare Tunel to access databag online.

**Comment by balzack** [comment]
@Stellarus37 I don't think google's push notification works with LineageOS, let me know if I am wrong. To get push notifications working in LineageOS you can install the ntfy app, which will enable UnifiedPush, which is also supported in Databag.

**Comment by Stellarus37** [comment]
I tested and it seems working 

![Screenshot_20250102-185333_Trebuchet.png](https://github.com/user-attachments/assets/a8131e55-970f-4036-a602-5ffa840ae5d1)

**Comment by mrnswetland** [comment]
> @Stellarus37 I don't think google's push notification works with LineageOS, let me know if I am wrong. To get push notifications working in LineageOS you can install the ntfy app, which will enable UnifiedPush, which is also supported in Databag.

Was this in response to my issue as well? I'm just using the stock android os

**Comment by mrnswetland** [comment]
I'm running Databag in Docker and using a reverse proxy with a Cloudflare domain.

**Comment by balzack** [comment]
@mrnswetland I agree that this should work on your stock android OS. I purchased an A24 and hopefully it will have arrived by the time I return from holidays. I hope to be able to reproduce the issue.

**Comment by mhbdesignr** [comment]
> Hi @Balzack, I'm MHB. I am a UI/UX and Visual Designer. I found out this project on contribute.design, and I would love to contribute to this project, but I am new to GitHub and would be nice if I had **directions/instructions** on what to do.

**Comment by balzack** [comment]
Can you retest when you get a chance? I resolved an issue that may have affected you.

**Comment by mrnswetland** [comment]
It works!

**Comment by balzack** [comment]
awesome! sorry it took so long to figure out on my end :(

**Comment by mrnswetland** [comment]
No worries! It's an amazing service, so I'm glad it got fixed

**Comment by martind2000** [comment]
Hi, I'm experiencing this same issue on a Jelly Max running Android 14 and the latest from FDroid, and Google Play on a Moto G. Any thoughts?

**Comment by balzack** [comment]
With FDroid, I think that means you installed the app from the FDroid store, correct?

Assuming that is the case, the only option for push notifications is to use UnifiedPush.

**Comment by martind2000** [comment]
Yes, installed from the FDroid store,  however the other phone was installed via the Google App store. How does it work via UnifiedPush, what needs to be done to get that to work?

**Comment by balzack** [comment]
Depending on how you want to configure it, it might be as easy as just installing the ntfy app from the fdroid store. Once you have ntfy installed, you will need to log out/in of your databag account to enable unifiedpush notifiications.

You can get more info here: https://unifiedpush.org/. You might want to change the background run settings for ntfy or you might also want to self host the relay server.
